packages/shared/src/components/modals/SteppedModal.tsx

```diff
@@ -173,7 +173,7 @@ function SteppedModal({
       overlayClassName="py-10"
     >
       <CloseButton
-        className="top-2 right-2 z-1"
```

Comment:
@rebelchris Fixing the alignment might break other stuff, since this modal will be replaced anyways, I recommend we just merge this as it is.

Reply:
I see, ok let's leave that part out of scope for now :)